### ISSUE TITLE
File types changed and will not open and search not finding anything

### ISSUE THREAD

**Issue by pickleton89** [MohrJonas/obsidian-ocr#36]
A number of my png files in my attachment folders have .hocr added to name of file. They will not open now. Additionally, when I invoke the OCR search window, it doesn't find anything. I see when opening Obsidian that the indexing counter finishes.

**Comment by MohrJonas** [comment]
Thank you for your issue. Can you confirm that you're using the newest version of obsidian-ocr, which is currently 2.0.0?

**Comment by pickleton89** [comment]
Yes. Everything thing is up to date. Also my obsidian is Version 1.1.12 (Installer 1.1.9)

**Comment by MohrJonas** [comment]
Alright, thanks for checking.
When you say the files have hocr added to the name, do you mean you have a file `x.png` and also a file `x.png.hocr`?

**Comment by pickleton89** [comment]
I sorted the files alphabetically and see that there is the base .png files and then another file, with the same name with the x.phg.hocr. I hadn’t noticed that before.

Jeff

**Comment by MohrJonas** [comment]
Alright, it's good to hear that the original file is still there, was a bit scared that I screwed something up, and it deleted the files 😌
The .hocr files are remnants of an older version of obsidian-ocr that stored the OCR information for `x.png` in `x.png.hocr`.
You can either leave them there and ignore them, or simply delete them.
Since version 2.0.0, the information is stored in a SQLite database, called `.obsidian-ocr.sqlite`, in the root of your vault.

Concerning the problem you described above:
Could you please open the developer console and see if any errors are reported?

**Comment by pickleton89** [comment]
I opened the OCR search command and typed in a query and got this error in console after pressing return on the search.



￼

**Comment by MohrJonas** [comment]
Unfortunately, I can't seem to see the attached image

**Comment by pickleton89** [comment]
<img width="586" alt="Screenshot of Obsidian (2-5-23, 1-50-49 PM)" src="https://user-images.githubusercontent.com/17555063/216848081-e141cdf3-2a55-42b0-bce8-b1d76c0eae18.png">

**Comment by pickleton89** [comment]
Sorry about that was replying by email and image didn't come through. I posted it above.

**Comment by MohrJonas** [comment]
Alright, thanks for the image.
Could you please enable `Log to file` in the settings of obsidian-ocr, restart obsidian and perform the same steps you did to produce the error above.
After that, could you please attach the log file.

**Comment by pickleton89** [comment]
Sorry, but I a not sure were the the log file gets created and how to find it.

**Comment by pickleton89** [comment]
I found the file. 
[obsidian-ocr.log](https://github.com/MohrJonas/obsidian-ocr/files/10677091/obsidian-ocr.log)

**Comment by MohrJonas** [comment]
Thank you for the log. I think I have somewhat of an idea what's going on here.
Could you please tell me which os you're using?

**Comment by pickleton89** [comment]
I am currently running macOS Ventura 13.2

**Comment by MohrJonas** [comment]
Okay, and could you tell me the output of `tesseract -v` in your terminal?

**Comment by pickleton89** [comment]
tesseract 5.2.0
 leptonica-1.82.0
  libgif 5.2.1 : libjpeg 8d (libjpeg-turbo 2.1.3) : libpng 1.6.39 : libtiff 4.4.0 : zlib 1.2.11 : libwebp 1.2.4 : libopenjp2 2.5.0
 Found NEON
 Found libarchive 3.6.2 zlib/1.2.11 liblzma/5.2.9 bz2lib/1.0.8 liblz4/1.9.4 libzstd/1.5.2
 Found libcurl/7.86.0 SecureTransport (LibreSSL/3.3.6) zlib/1.2.11 nghttp2/1.47.0
